client/src/app/pages/issues/affected-applications/affected-applications.tsx

```diff
@@ -60,7 +64,7 @@ export const AffectedApplications: React.FC = () => {
       name: "Name",
       description: "Description",
       businessService: "Business service",
-      effort: "Total Effort",
+      effort: "Effort",
```

Comment:
Humm .. I think this change is causing this e2e test fail:

https://github.com/konveyor/tackle2-ui/actions/runs/8843230714/job/24306934573?pr=1846#step:10:1817

Reply:
Ah right - I think a change recently went in to change to total effort on e2e side 😵‍💫

Reply:
![Screenshot from 2024-04-29 13-51-52](https://github.com/konveyor/tackle2-ui/assets/3985964/e05cc185-e75e-44e3-886e-6e27041ab308)

Humm -- so the column name in the code is used as the `data-label` value.  There are two paths forward:
  1. Change the test to look at the new name "Effort"
  2. Change the effort value back to "Total Effort".  Since the heading text is being built manually now (to include the tooltip icon), this should still no cause any visual change to your change.

re: @kpunwatk